### PR TITLE
SCC Python SDK

### PR DESCRIPTION
**Pull request by pbvaishnavi**
## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [ ] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Does this PR introduce a breaking change?    
- [x] Yes
- [ ] No

### CODE REVIEW

File: ibm_scc/security_and_compliance_center_api_v3.py [hunk 1]
```diff
@@ -0,0 +1,13274 @@
...
+    :attr List[Implementation] assessments: (optional) The assessments.
+    """
+
+    def __init__(
+        self,
+        *,
+        control_specification_id: str = None,
+        responsibility: str = None,
+        component_id: str = None,
+        componenet_name: str = None,
```
Comment (tyao117) [inline ibm_scc/security_and_compliance_center_api_v3.py:5857]:
```suggestion
        component_name: str = None,
```

File: ibm_scc/security_and_compliance_center_api_v3.py [hunk 2]
```diff
@@ -0,0 +1,13274 @@
...
+
+class ControlSpecifications:
+    """
+    The control specifications of a control library.
+
+    :attr str control_specification_id: (optional) The control specification ID.
+    :attr str responsibility: (optional) The responsibility for managing the
+          control.
+    :attr str component_id: (optional) The component ID.
+    :attr str componenet_name: (optional) The component name.
```
Comment (tyao117) [inline ibm_scc/security_and_compliance_center_api_v3.py:5843]:
```suggestion
    :attr str component_name: (optional) The component name.
```

File: ibm_scc/security_and_compliance_center_api_v3.py [hunk 3]
```diff
@@ -0,0 +1,13274 @@
...
+    ) -> None:
+        """
+        Initialize a ControlSpecifications object.
+
+        :param str control_specification_id: (optional) The control specification
+               ID.
+        :param str responsibility: (optional) The responsibility for managing the
+               control.
+        :param str component_id: (optional) The component ID.
+        :param str componenet_name: (optional) The component name.
```
Comment (tyao117) [inline ibm_scc/security_and_compliance_center_api_v3.py:5871]:
```suggestion
        :param str component_name: (optional) The component name.
```

File: ibm_scc/security_and_compliance_center_api_v3.py [hunk 4]
```diff
@@ -0,0 +1,13274 @@
...
+        :param str environment: (optional) The control specifications environment.
+        :param str control_specification_description: (optional) The control
+               specifications description.
+        :param int assessments_count: (optional) The number of assessments.
+        :param List[Implementation] assessments: (optional) The assessments.
+        """
+        self.control_specification_id = control_specification_id
+        self.responsibility = responsibility
+        self.component_id = component_id
+        self.componenet_name = componenet_name
```
Comment (tyao117) [inline ibm_scc/security_and_compliance_center_api_v3.py:5881]:
```suggestion
        self.component_name = component_name
```

**Review by tyao117 — CHANGES_REQUESTED** [review]
please change `componenet_name` to `component_name`

File: ibm_scc/security_and_compliance_center_api_v3.py (hunk 1 above)
Comment (pbvaishnavi) [reply ibm_scc/security_and_compliance_center_api_v3.py]: updated

File: ibm_scc/security_and_compliance_center_api_v3.py (hunk 4 above)
Comment (pbvaishnavi) [reply ibm_scc/security_and_compliance_center_api_v3.py]: Resolved.

**Comment by archanaponnada** [comment]
Closing this as the changes under these PR are covered under PR#45

**Pull request closed without merging**